src/renderer/components/pages/Config/pages/ArmPresetsConfig/ArmPreset.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useCallback } from 'react';
+import {
+  ArmPreset,
+  armPresetsSlice,
+} from '@/renderer/store/modules/armPresets';
+import { styled } from '@/renderer/globalStyles/styled';
+import { LabeledInput } from '@/renderer/components/common/LabeledInput';
+import { useDispatch } from 'react-redux';
+import { FaTimes } from 'react-icons/fa';
+import ArmJointInput from './ArmJointInput';
+
+interface ArmPresetProps {
+  preset: ArmPreset;
+}
+
+//Temporary UI limit for each joint value.
+interface JointBoundaries {
+  min: number;
+  max: number;
+}
+
+const jointBoundaries: JointBoundaries[] = [
```

Comment:
Nice feature, is it to prevent invalid entries  ?

Reply:
Yes, also to prevent some joints to be moved to impossible configurations, for example the joint 3 below 95 degrees could collide with the arm itself.